src/libfido2Config.cmake.in

```diff
@@ -0,0 +1,10 @@
+set(FIDO_VERSION @FIDO_VERSION@)
```

Comment:
Is it canonical to set a variable like this? I was under the impression that this was already handled by `write_basic_package_version_file()` (albeit under a slightly different variable name).

Reply:
yes, `write_basic_package_version_file` provides it under a different name, so I did define this. can change it though.